configs/stash-origin/overrides/xrootd/10-stash-origin-overrides.conf

```diff
@@ -0,0 +1,3 @@
+[Unit]
```

Comment:
We should either say "see comments in stash-origin-overrides.conf" or duplicate those comments here.

Reply:
Fixed.